### DISCUSSION TITLE
Disable automatic subscription to discussions

### DISCUSSION THREAD

**Discussion by inxomnyaa** [community/community#2898] [Discussions] [answered]
![image](https://user-images.githubusercontent.com/8733998/101634378-e3ce8f00-3a28-11eb-8f84-dd099e5d34af.png)
My problem is that these keep popping up and polluting my inbox.
I am only subscribed to releases to turn down the heat.
Since one is automatically subscribed to the discussions, one has to unsubscribe on every repo that enables this feature.

**Comment by becca** [comment, marked as answer]
👋 hello there! I believe right now you might be watching those repositories. You can unsubscribe from Discussions specific activity by using the 🆕 custom notification controls

* Changelog entry: https://github.blog/changelog/2020-11-20-custom-notification-controls/
* Documentation on the feature: https://docs.github.com/en/free-pro-team@latest/github/managing-subscriptions-and-notifications-on-github/configuring-notifications

Once you reconfigure some of your current `watch` settings to be more granular you will no longer receive Discussions notifications, should you decide to disable those as a part of your updates. 🙇
